scripts/setup/install_bookrunner_deps.sh

```diff
@@ -5,18 +5,17 @@
 set -eu
 
 # The book runner report is generated using [Litani](https://github.com/awslabs/aws-build-accumulator)
+FILE="litani-1.22.0.deb"
+URL="https://github.com/awslabs/aws-build-accumulator/releases/download/1.22.0/$FILE"
 
-# Litani's dependencies:
-DEPS=(
-  gnuplot # Not required but recommended
-  graphviz
-)
+set -x
```

Comment:
This can be included with line 5 (`set -eux`).

Reply:
I kept it like this to be consistent with other scripts like [THIS](https://github.com/model-checking/kani/blob/main/scripts/setup/ubuntu/install_cbmc.sh) one.
Should i update?

Reply:
Not sure why the other script is done this way, but feel free to skip making the change then.